jsx/DataTable.js

```diff
@@ -552,7 +568,7 @@ class DataTable extends Component {
     );
 
     return (
-      <div style={{margin: '14px'}}>
+      <div name={this.props.name} style={{margin: '14px'}}>
```

Comment:
"name" in HTML is for attaching a name to submit data from an HTML form as. I'm not sure what it means to add it to a div.

Reply:
i think i can get rid of this. good catch!